Examples/MAX32655/BLE_otac/datc_main.c

```diff
@@ -768,9 +771,12 @@ static void datcBtnCback(uint8_t btn)
                 /* Start the WDXC data stream */
                 datcCb.blockOffset[connId - 1] = BLOCK_OFFSET_INIT;
 
-                /* Put file request */
-                WdxcFtcSendPutReq(connId, datcCb.fileList[connId - 1][0].handle, BLOCK_OFFSET_INIT,
-                                  FILE_SIZE, BLOCK_OFFSET_INIT + FILE_SIZE, 0);
+                /* Put file request only if file has been discovered*/
+                if (fileDiscovered)
+                    WdxcFtcSendPutReq(connId, datcCb.fileList[connId - 1][0].handle,
```

Comment:
clang-format should catch this, but you'll need braces around the body of this 'if' block.